python/ray/serve/controller.py

```diff
@@ -487,6 +488,12 @@ def get_http_config(self):
             return None
         return self.http_proxy_state_manager.get_config()
 
+    def get_grpc_config(self):
+        """Return the gRPC proxy configuration."""
+        if self.http_proxy_state_manager is None:
```

Comment:
as a follow up, we should rename the http proxy state manager.

Reply:
Yes, I'm fully aware with will the name changes :)